api/Element.json

```diff
@@ -1448,6 +1449,110 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "width_and_height": {
+          "__compat": {
+            "description": "<code>width</code> and <code>height</code> properties",
+            "support": {
+              "webview_android": {
+                "version_added": null
```

Comment:
Both android flavors should be `true`.

Also, why aren't these separate entries?

Reply:
I think(?) any implementation with width also has height (ditto for x/y), and this is just based on the way the table was built in the old page. I can see an argument for separating them, but we usually don't list every possible property of subfeatures anyway, so I'm not sure if we have a strong reason to include these separately.

@Elchi3 any opinion here?

Reply:
I agree to having separate entries as they are more easily machine-readable. Please change this here and for x_and_y.